libgap.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: @PACKAGE_NAME@
```

Comment:
Since the package name is static I would suggest to manually expand this to make clearer what happens -- unless you have specific reasons to use `@PACKAGE_NAME@` here? I.e.
```suggestion
Name: GAP
```
(Based on what I read about pkg-config, this file seems to be for human eyes only, so using "GAP" instead of "gap" should be indeed fine?)

Reply:
I'd say `@PACKAGE_NAME@` is more future-proof.

Reply:
You mean in case we decide to rename GAP to Frobnicator2000 ? 😂

Reply:
If that happens and we unexpectedly loose (very unlikely under US or German law) then I am sure we'll be delighted about this time saver 😜.